wordpress-mobile/WordPress-Android
# [Site Creation] Step 1: Remove verticals site creation step screen

This PR removes verticals site creation screen as part of #11420 and #11421 and is one of multiple steps to remove verticals step (Step 1 of 3) from site creation flow.

P.S.  Review after https://github.com/wordpress-mobile/WordPress-Android/pull/11434 is merged to the Master PR.

**To test:**

- Go to My site
- Click Switch Site
- Click "+" button to launch Add new site
- Click Create WordPress.com site on Add new site
- Choose a segment Blog/Business/Professional/Blank Canvas to open Step 1 
- Verify 
    - Verticals site creation step is not displayed
    - Site info step is displayed as Step 1 of 2

![vertical_step_not_displayed](https://user-images.githubusercontent.com/1405144/76498969-e41f0700-6463-11ea-9b2a-cbccc731cb9e.gif)

PR submission checklist:
